template/carousel/carousel.html

```diff
@@ -1,6 +1,6 @@
 <div ng-mouseenter="pause()" ng-mouseleave="play()" class="carousel" ng-swipe-right="prev()" ng-swipe-left="next()">
     <ol class="carousel-indicators" ng-show="slides.length > 1">
-        <li ng-repeat="slide in slides track by $index" ng-class="{active: isActive(slide)}" ng-click="select(slide)"></li>
+        <li ng-repeat="slide in slides | orderBy:'index' track by $index" ng-class="{active: isActive(slide)}" ng-click="select(slide)"></li>
```

Comment:
Sorry, but I'm not sure that I understand what you mean. In this peace of code we're just ordering the carousel indicators by the slide index parameter. Do you mean that we should do something like this:
```html
<li ng-repeat="slide in slides | orderBy:orderIndicators track by $index" ng-class="{active: isActive(slide)}" ng-click="select(slide)"></li>
```
and then in controller:
```html
$scope.orderIndicators = function(slide) {
    return slide.index;
};
```
What's the purpose of such extension?